README.md

```diff
@@ -83,6 +83,15 @@ access to the site, use `--ssh` to override the protocol:
 $ basher install --ssh juanibiapina/gg
 ~~~
 
+### Installing a local package
+
+If you develop a package locally and want to try it through basher,
+use the `link` command:
+
+~~~ sh
+$ basher link my_package my_namespace/my_package
```

Comment:
the first argument should say `directory` here, right?

Reply:
Indeed, would be clearer

Reply:
Done